### PR TITLE
kubevirt: ssp: Add CI for branch "release-v0.22"

### PR DESCRIPTION
**Pull request by akrejcir**
Added CI config for new branch: `release-v0.22`

### CODE REVIEW

File: ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.22.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
+base_images:
+  metrics-linter:
+    name: prom-metrics-linter
+    namespace: ci
+    tag: v0.0.2
```
Comment (0xFelix) [inline ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.22.yaml:5]: Does this ever have to be updated?

**Review by 0xFelix — APPROVED** [review]
/lgtm

File: ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.22.yaml (hunk 1 above)
Comment (akrejcir) [reply ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.22.yaml:5]: Maybe @assafad will know? It was added in this PR: https://github.com/openshift/release/pull/41234

Comment (akrejcir) [reply ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.22.yaml:5]: I see Assaf doesn't work on this project anymore. Maybe @machadovilaca or @sradco would know?

**Comment by akrejcir** [comment]
/pj-rehearse pull-ci-kubevirt-ssp-operator-release-v0.22-e2e-single-node-functests

**Comment by akrejcir** [comment]
Upgrade tests are broken now. They will be fixed in the next release soon.

/pj-rehearse ack

File: ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.22.yaml (hunk 1 above)
Comment (akrejcir) [reply ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.22.yaml:5]: Please, when you have a moment @machadovilaca or @sradco , can you take a look? Or do you know who to ask about this?

Comment (avlitman) [reply ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.22.yaml:5]: Hi this need to be upgraded from time to time, like now: https://github.com/openshift/release/pull/72477

**Pull request merged**